core/spec/models/spree/calculator/default_tax_spec.rb

```diff
@@ -70,51 +68,145 @@
           # 60 - 57.14 = $2.86
           expect(calculator.compute(order).to_f).to eql 2.86
         end
+
+        context "when the order's tax address is outside the default VAT zone" do
+          let(:order_zone) { create(:zone, countries: [address.country]) }
+          let(:default_vat_country) { create(:country, iso: "DE") }
+
+          before do
+            rate.zone.update(countries: [default_vat_country])
+            # The order has to be reloaded here because of tax zone caching.
+            order.reload
+          end
+
+          it 'creates a negative amount, indicating a VAT refund' do
+            expect(subject.compute(order)).to eq(-2.86)
+          end
+        end
       end
     end
+  end
+
+  describe '#compute_line_item' do
+    it 'is an alias to #compute_item' do
+      expect(subject.method(:compute_line_item)).to \
+        eq(subject.method(:compute_item))
+    end
+  end
+
+  describe '#compute_shipment' do
+    it 'is an alias to #compute_item' do
+      expect(subject.method(:compute_shipment)).to \
+        eq(subject.method(:compute_item))
+    end
+  end
+
+  describe '#compute_shipping_rate' do
+    it 'is an alias to #compute_item' do
+      expect(subject.method(:compute_shipping_rate)).to \
+        eq(subject.method(:compute_item))
+    end
+  end
+
+  context 'when given a line item' do
+    let(:line_item) { Spree::LineItem.new }
+
+    it 'calls #compute_line_item' do
+      expect(subject).to receive(:compute_line_item)
```

Comment:
I don't like stubbing methods on the file that we're writing the spec for. I don't think that this test provides value.